content/en/blog/_posts/2021-12-15-pod-security-admission-beta.md

```diff
@@ -0,0 +1,612 @@
+---
+layout: blog
+title: 'Pod Security Admission (PSA) Graduates to Beta'
+date: 2021-12-15
+slug: pod-security-admission-beta
+---
+
+**Authors:** Jim Angel (Google), Lachlan Evenson (Microsoft)
+
+With the release of Kubernetes v1.23, [Pod Security Admission (PSA)](/docs/concepts/security/pod-security-admission/) has now entered beta. PSA is a [built-in](/docs/reference/access-authn-authz/admission-controllers/) admission controller that evaluates pod specifications against a predefined set of [Pod Security Standards](/docs/concepts/security/pod-security-standards/) and determines whether to `admit` or `deny` the pod from running. PSA is the successor to [PodSecurityPolicy](/docs/concepts/policy/pod-security-policy/) which was deprecated in v1.21 and is removed in v1.25. Let's explore how Pod Security Admission works in the hope that cluster administrators and developers alike will use this to enforce secure defaults for their workloads. In this blog we will cover the key concepts of Pod Security Admission along with how to use it. 
```

Comment:
nit: 
> will be removed in v1.25

Reply:
Good callout! I wonder if we should instead update `which was deprecated in v1.21` to `which is deprecated in v1.21` instead, to keep everything present tense (inline with the [style guide](https://kubernetes.io/docs/contribute/style/style-guide/#use-present-tense)).

However, not sure how strict that rule would be in this context (speaking as a blog, in a point of time, where one item is past tense and one is future tense) cc @sftim

Reply:
“_which was deprecated in v1.21_” is how I'd say it conversationally:
- A: “I'm using PodSecurityPolicy”
- B: “Kubernetes deprecated that in v1.21”
- A: “It was deprecated in v1.21? I'd better go and read about alternatives!”
- B: “Here's my rate card, let me know if you'd like help here.”

Reply:
With that same mindset, you think we should use `s/is removed in v1.25/will be removed in v1.25`?